content/en/docs/tasks/configure-pod-container/create-hostprocess-container.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Create a Windows HostProcess Container
```

Comment:
This feels slightly off message. How about “Create a Windows Pod That Has a HostProcess Container”, or something else that mentions creating a _Pod_.

Reply:
all containers must be hostprocess so something like "Create a Windows HostProcess Pod" or "Create a Windows  Pod as Hostprocess"